apps/browser/qml/pages/components/PermissionListItem.qml

```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+ListItem {
+    id: root
+
+    property alias label: valueButton.label
+    property alias value: valueButton.value
+    property alias iconSource: icon.source
+
+    width: parent.width
+    contentHeight: Theme.itemSizeMedium
+    onClicked: openMenu()
+
+    Row {
+        width: parent.width - 2*Theme.horizontalPageMargin
+        x: Theme.horizontalPageMargin
+        anchors.verticalCenter: parent.verticalCenter
+
+        Icon { id: icon }
```

Comment:
The Icon and ValueButton look misaligned vertically to me. The should both be vertically centred in the parent row.